contracts/interfaces/IMain.sol

```diff
@@ -114,43 +114,43 @@ interface IAuth is IAccessControlUpgradeable {
 interface IComponentRegistry {
     // === Component setters/getters ===
 
-    event RTokenSet(IRToken indexed oldVal, IRToken indexed newVal);
+    event RTokenSet(IRToken oldVal, IRToken newVal);
```

Comment:
Huh, looks like these events are leftover from a time where it was possible to change the addresses, which is no longer the case. I think it's probably not worth changing, though. You could imagine an RToken upgrades Main to include the ability to change the component addresses, so maybe it's worth keeping around. And the cost is basically zero.